x/group/keeper/keeper.go

```diff
@@ -443,3 +450,30 @@ func (k Keeper) TallyProposalsAtVPEnd(ctx sdk.Context) error {
 	}
 	return nil
 }
+
+// assertMetadataLength returns an error if given metadata length
+// is greater than defined MaxMetadataLen in the module configuration
+func (k Keeper) assertMetadataLength(metadata, description string) error {
+	if metadata != "" && uint64(len(metadata)) > k.config.MaxMetadataLen {
```

Comment:
The `metadata != ""` is redundant given that len(metadata) will either be 0 or greater than 0